pkg/util/printer/printer.go

```diff
@@ -84,6 +84,10 @@ func GetTiDBInfo() string {
 		config.GetGlobalConfig().Store,
 		enterpriseVersion,
 	)
+	if config.IsCloudStore() {
+		info += "\nNext Generation"
```

Comment:
the output is a bit different with other line as the format `{name}: {value}`

```
Next Generation: {some API version}
```

is better?